NON_ENG.md

```diff
@@ -1 +1,2 @@
 - [GBA develop Wiki](http://akkera102.sakura.ne.jp/gbadev/) - Japanese wiki including many documentations, tutorials and source codes (especially an AVG game engine).
+- [GBAで動画再生](http://tech.g1.xrea.com/notes/gba-music.htm) - A summary of video codec and players on GBA.
```

Comment:
This list targets (native, not auto/machine translated) english content, so we can't accept this kind of links here. We could, however create additional file(s) to group resources in other languages (this may depend on our availability of reviewers in those languages..)

Reply:
isn't it `NON_ENG.md`?